tslint.json

```diff
@@ -14,13 +14,7 @@
         false
       ],
       "no-console": {
-        "severity": "warning"
-      },
-      "max-line-length": {
-        "options": {
-          "limit": 120
```

Comment:
I think we should retain line length.

Reply:
I agree too. No need to remove it

Reply:
samesies

Reply:
Isn't this part of `tslint:recommended` https://github.com/palantir/tslint/blob/237fd905a9fb3c9ca9325d2da6e8145e4a44fe51/src/configs/recommended.ts#L63